File: pset6/mario/less/mario.py
from cs50 import get_int


def main():
    i = get_positive_int()

    r = '#'
    s = ' '
    for a in range(i):
        print(f"{s * (i - a - 1)}{r * (a + 1)}")


def get_positive_int():
    while True:
        n = get_int("Enter positive number: ")
        if n <= 8 and n > 0:
            break
    return n


main()

File: pset1/mario/more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;

    do
    {
        n = get_int("Enter positive number: ");
    }
    while (n < 1 || n > 8);

    for (int i = 0; i < n; i++)
    {
        for (int b = 0; b < n - i - 1; b++)
        {
            printf(" ");

        }
        for (int a = 0; a < i + 1; a++)
        {
            printf("#");
        }

        printf("  ");

        for (int a = 0; a < i + 1; a++)
        {
            printf("#");
        }

        printf("\n");

    }
}